src/js/models/v2/agency/BaseAgencySubcomponentsList.js

```diff
@@ -5,11 +5,13 @@
 import { formatMoneyWithUnitsShortLabel } from 'helpers/moneyFormatter';
 
 const BaseAgencySubcomponentsList = {
-    populate(data) {
-        this.id = data?.id || data?.code || '';
+    populate(data, id, level5) {
+        this.id = id || data?.id || data?.code || '';
         this.name = data?.name || '';
+        if (!level5) {
```

Comment:
should the changes to this file be reverted because we're not doing level 5 now?

Reply:
I could go either way on this one - i left in the redux actions and the functions to get data for level 5, since we're going to use them soon and they aren't affecting anything. i think we could say the same about this, but it's also small enough to remove it now and easily replace it later.

Reply:
i think we should review it and think through the level 5 implementation more with the ticket next sprint

Reply:
ok, sounds good.
so remove this now?

Reply:
yep, please remove it now

Reply:
removed